docs/introduction.asciidoc

```diff
@@ -27,7 +28,7 @@ In order to connect transactions with related spans and errors, and propagate tr
 the agent relies on Go's built-in https://golang.org/pkg/context/[context] package:
 transactions and spans are stored in context objects.
 For example, for incoming HTTP requests, in-flight trace data will be recorded in the `context` object accessible through
-https://golang.org/pkg/net/http/#Request.Context[net/http.Context].
+https://golang.org/pkg/net/http/#Request.Context[net/http.Context] / https://pkg.go.dev/github.com/valyala/fasthttp#RequestCtx[fasthttp.RequestCtx].
```

Comment:
Please revert this change.

Users of fasthttp should know that it doesn't use `context.Context`, and can refer to the apmfasthttp section for specifics. `net/http` is mentioned because it is most common. It might be worth extending the code example under apmfasthttp to show that you can pass a RequestCtx into apm.TransactionFromContext.